_includes/master/reqs-sys.md

```diff
@@ -0,0 +1,31 @@
+## Node hosts
+
+- AMD64 processor
+
+- Linux kernel 3.10 or later. The following distributions are known to work 
+  well with {{site.prodname}} and {{include.orch}}.
+  - RedHat Linux 7{% if include.orch == "Kubernetes" or include.orch == "host protection" %}
+  - CentOS 7
+  - CoreOS Container Linux stable
+  - Ubuntu 16.04
+  - Debian 8
+  {% endif %}{% if include.orch == "OpenShift" %}
+  - CentOS 7
+  {% endif %}{% if include.orch == "OpenStack" %}
+  - Ubuntu 16.04
+  - CentOS 7
+  {% endif %}
+{% if include.orch == "Kubernetes" %}
+<br>
+- No other pod networking configurations stored in `/etc/cni/net.d`.
```

Comment:
I find the weaving of orchestrator specific requirements through this doc really confusing. Can we not put the orchestrator specific stuff in the per-orchestrator docs and only include the generic requirements in this one? Like, why is this here but the kube-proxy settings are in the other doc?

Reply:
I know it is a little odd but to me, I like having them all in the one file as you can easily see the differences between the orchestrators. At least to me, it makes sense. But there is definitely a pretty good argument to be made that this content is sufficiently unique on a per orchestrator basis that we shouldn't even use an include. I have been debating that, as well. The `/etc/cni/net.d` is just here because it is part of this one section and the kube-proxy is in a new section.

Reply:
We can always iterate more, and I think further changes will throw more light on this question as we continue improving our doc structure.  So, unless there's already a clear answer, I would suggest not worrying about this point for this PR.